### DISCUSSION TITLE
[@types/rtlcss] uses the wrong types for the "options" parameters to .process() and .configure()

### DISCUSSION THREAD

**Discussion by catrope** [DefinitelyTyped/DefinitelyTyped#59399] [Issues with a @types package]
The `rtlcss.process()` method takes a parameter called `options`, which is currently typed as just `object`. However, the type definitions do include a type for this called `ConfigOptions`, which is used for the `.configure()` method. This type should be used here instead.

Also, even though `ConfigOptions` is the right type for the first parameter to `.process()`, it's the wrong type for the `.configure()` method's sole parameter. According to its own documentation comment, that method takes "an object containing RTLCSS options, plugins and hooks". That's ambiguous, but [[https://github.com/MohammadYounes/rtlcss/blob/0e9d9b384356ca277f141ea41666ba86bdece5a0/lib/rtlcss.js#L234this code]].
makes clear that its parameter is an object with the keys `options`, `plugins` and `hooks`, which each correspond to the parameters of the `rtlcss()` function. So the correct type would be something more like `{ options: ConfigOptions, plugins: PluginOptions, hooks: HookOptions }` (where `PluginOptions` is a type that doesn't exist, but maybe should).

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
see #59439
